docs/source/tutorial/tutorial-define-additional-mutations.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: "12. Define additional mutations"
+---
+
+In this section you will implement additional mutations for booking and cancelling trips.
+
+## Add the `BookTrip` mutation
+
+In [Sandbox](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fapollo-fullstack-tutorial.herokuapp.com), open the Schema tab by clicking its icon, select the `Mutations`, and take a look at the `bookTrips` mutation:
+
+<img alt="The docs for book trips" class="screenshot" src="images/sandbox_schema_book_trips.png"/>
+
+Click the play button to the right to open this mutation in Explorer. Click the plus button to add the `bookTrips` mutation:
+
+<img alt="The book trips operation immediately after adding it" class="screenshot" src="images/explorer_book_trips_starter.png"/>
+
+You can see in the left sidebar that this takes an argument of an array of IDs (which was added as `$bookTripsLaunchIds`), and the object returned from the operation has three properties:
+
+* A `success` boolean indicating whether the booking succeeded
+* A `message` string to display to the user
+* A list of `launches` the current user has booked
+
+Click the plus signs next to `success` and `message` to add those to your operation.
+
+In the "Variables" section of Sandbox Explorer, add an array of identifiers. In this case, we'll use a single identifier to book one trip:
+
+```json title="(Sandbox Explorer)"
+{"bookTripsLaunchIds": ["25"]}
+```
+
+Next, directly next to the word "Variables", you'll see the word "Headers". Click that to bring up the headers section. Click the "New Header" button, and add "Authorization" in the header key text box and paste the token you got back in the last section for the value:
+
+<img alt="The headers section" class="screenshot" src="images/explorer_add_auth_header.png"/>
+
+Now, click the Submit Operation button to run your authorized query. You'll get back information regarding the trips (or in this case, trip) you've just booked.
+
+> Note: If you receive an error that says "Cannot read property 'id' of null", that means your user was not found based on the token you passed through. Make sure your authorization header is properly formatted and that you're actually logged in!
+
+<img alt="Explorer showing the result of booking a trip with an array of IDs" class="screenshot" src="images/explorer_book_with_trip_ids.png"/>
+
+With a mutation written like this, you can book any number of trips you want at the same time. However, the booking mechanism in our application will only let you book one trip at a time.
+
+Luckily, there's an easy way to update the mutation so it's required to only take a single object. First, update the name of your operation in Explorer to the singular `BookTrip` (and remove `Mutation` since that will be added for us by code generation). Next, update the mutation to take a single `$id`, then pass an array containing that `$id` to the `bookTrips` mutation:
```

Comment:
~(and remove `Mutation` since that will be added for us by code generation)~ - this is implementation detail they really don't need to be concerned about. I believe we check for the word "Mutation" and append it if it's not there but either way if we have a bug that results in "BookTripsMutationMutation" then we should fix it.

tl;dr - this isn't tutorial-level detail that helps them learn Apollo iOS.